node/src/main/scala/coop/rchain/node/api/DeployGrpcServiceV1.scala

```diff
@@ -291,5 +295,21 @@ object DeployGrpcServiceV1 {
             }
           )
           .flatMap(Observable.fromIterable)
+
+      def status(request: com.google.protobuf.empty.Empty): Task[StatusResponse] =
+        (for {
+          address <- RPConfAsk[F].ask
+          peers   <- ConnectionsCell[F].read
+          nodes   <- NodeDiscovery[F].peers
+          status = Status(
+            version = VersionInfo(api = 1.toString, node = coop.rchain.node.web.VersionInfo.get),
+            address.local.toAddress,
+            networkId,
+            shardId,
+            peers.length,
+            nodes.length
```

Comment:
What is the difference between nodes and peers actually?

Reply:
Peers are maintained by ConnectionsCell and nodes are managed by Kademlia. That should mean that peers are actively connected nodes.